WooCommerce/WooCommerceUITests/Tests/OrdersTests.swift

```diff
@@ -0,0 +1,20 @@
+import UITestsFoundation
+import XCTest
+
+final class OrdersTests: XCTestCase {
+    override func setUp() {
```

Comment:
Just a line spacing nitpick, for consistency:

```suggestion
final class OrdersTests: XCTestCase {

    override func setUp() {
```